<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNSS_Multi_Threading</class>
 <widget class="QDialog" name="GNSS_Multi_Threading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBoxNumberThreads">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_33">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonNumberThreadsOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonNumberThreadsCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonNumberThreadsDefault">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Set Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Set number of threads:&lt;/b&gt;&lt;br&gt;&lt;br&gt;Set the number of threads to use during gLAB processing. Multiple threads allow to parallelize computation load, but note that only the cycle-slip and model computation are run in multi-threading mode.&lt;br&gt;There must be as many cores as threads, so each thread run on a different core.&lt;br&gt;The default is to run with a single thread.&lt;br&gt;&lt;br&gt;&lt;b&gt;NOTE:&lt;/b&gt; In gLAB, the single thread and multi-thread modes are separated in different command line executables (with the same source code). The names of the multi-thread executables are the following:&lt;ul&gt;&lt;li&gt;&lt;b&gt;Windows:&lt;/b&gt; gLAB_MultiThread.exe&lt;/li&gt;&lt;li&gt;&lt;b&gt;Linux:&lt;/b&gt; gLAB_MultiThread_linux&lt;/li&gt;&lt;li&gt;&lt;b&gt;Mac:&lt;/b&gt; gLAB_MultiThread_mac&lt;br&gt;&lt;/li&gt;&lt;/ul&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Set number of threads:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxNumberThreads</tabstop>
  <tabstop>pushButtonNumberThreadsDefault</tabstop>
  <tabstop>pushButtonNumberThreadsCancel</tabstop>
  <tabstop>pushButtonNumberThreadsOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
